### DISCUSSION TITLE
🚀 Feature: Umgang mit umbrechenden Labels bei Inputfeldern verbessern

### DISCUSSION THREAD

**Discussion by jschendzielorz** [public-ui/kolibri#6918] [Ideas]
### Goals

Labels sollen bei zu langem Text oder zu kleiner Auflösung das Layout eines Formulars nicht kaputt machen.

### Background

Wenn Labels in Formularen umbrechen, weil sie zu lang oder die Auflösung zu klein ist, bricht das das Layout des Formulars, weil Inputfelder nach unten rutschen.

### Proposal

1. Labels von Inputfelder per `text-overflow: ellipsis` abschneiden
2. Labeltext bei hover vollständig anzeigen

**Comment by deleonio** [comment]
Hallo @jschendzielorz,

vielen Dank für den Hinweis. Aus Sicht der Barrierefreiheit dürfen keine Inhalte mit Ellipsis abgeschnitten werden. Daher ist es redaktionell wichtig, kurze und klare Beschriftungen zu verwenden.

Neben einem Hover müsste man auch den Tastaturfokus beachten und das Label wohl antabbar machen.

Sehr gerne können weitere Argumente hierzu ausgetauscht werden.

**Comment by deleonio** [comment]
@AntnSaj könnt Ihr Euch bitte diese Frage anschauen und uns Feedback geben?
